tests/simple/testdata/wrappers.textproto

```diff
@@ -0,0 +1,260 @@
+    name: "to_json_string"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_value: google.protobuf.Int64Value{value: 9223372036854775807}}.single_value"
+    value {
+      string_value: "9223372036854775807"
+    }
+  }
+}
+
+# google.protobuf.UInt32Value
+section {
+  name: "uint32"
+  test {
+    name: "to_any"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_any: google.protobuf.UInt32Value{value: 1u}}.single_any"
+    value {
+      uint64_value: 1
+    }
+  }
+  test {
+    name: "to_json"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_value: google.protobuf.UInt32Value{value: 1u}}.single_value"
+    value {
+      double_value: 1
+    }
+  }
+}
+
+# google.protobuf.UInt64Value
+section {
+  name: "uint64"
+  test {
+    name: "to_any"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_any: google.protobuf.UInt64Value{value: 1u}}.single_any"
+    value {
+      uint64_value: 1
+    }
+  }
+  test {
+    name: "to_json_number"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_value: google.protobuf.UInt64Value{value: 1u}}.single_value"
+    value {
+      double_value: 1
+    }
+  }
+  test {
+    name: "to_json_string"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_value: google.protobuf.UInt64Value{value: 18446744073709551615u}}.single_value"
+    value {
+      string_value: "18446744073709551615"
+    }
+  }
+}
+
+# google.protobuf.FloatValue
+section {
+  name: "float"
+  test {
+    name: "to_any"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_any: google.protobuf.FloatValue{value: 1.0}}.single_any"
+    value {
+      double_value: 1
+    }
+  }
+  test {
+    name: "to_json"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_value: google.protobuf.FloatValue{value: 1.0}}.single_value"
+    value {
+      double_value: 1
+    }
+  }
+}
+
+# google.protobuf.DoubleValue
+section {
+  name: "double"
+  test {
+    name: "to_any"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_any: google.protobuf.DoubleValue{value: 1.0}}.single_any"
+    value {
+      double_value: 1
+    }
+  }
+  test {
+    name: "to_json"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_value: google.protobuf.DoubleValue{value: 1.0}}.single_value"
+    value {
+      double_value: 1
+    }
+  }
+}
+
+# google.protobuf.BytesValue
+section {
+  name: "bytes"
+  test {
+    name: "to_any"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_any: google.protobuf.BytesValue{value: b'foo'}}.single_any"
+    value {
+      bytes_value: "foo"
+    }
+  }
+  test {
+    name: "to_json"
+    container: "google.api.expr.test.v1.proto3"
+    expr: "TestAllTypes{single_value: google.protobuf.BytesValue{value: b'foo'}}.single_value"
```

Comment:
As a follow up PR, could you enable encoders and verify that the following is true?

```
TestAllTypes{single_value: google.protobuf.BytesValue{value: b'foo'}}.single_value == base64.encode("foo") 
```